### ISSUE TITLE
[0.9.0 Staging-1494] If you 'x' out of creating a Work Party, you can't post new Work Parties

### ISSUE THREAD

**Issue by ThePiachu** [StrangeLoopGames/EcoIssues#15541]
After #15540, I pressed "x" on the Work Party creation screen. When I tried creating a new Work Party, I got this error:

![image](https://user-images.githubusercontent.com/964559/78419877-e4439a00-75fe-11ea-82e5-44e30ab41960.png)

I don't see any way for me to resume creating that other Work Party, so it's a soft-lock on that functionality for me. The problem even persists after restarting the game...

**Comment by beastyn** [comment]
Work tab on the right side should store your unfinished parties and contracs

**Comment by ThePiachu** [comment]
Would be nice to have it as a part of the job board UI.

**Comment by ThePiachu** [comment]
I had to explain this issue to 3-4 other players 3 days into the new test server, so it might be an UX problem worth revisiting.

**Comment by ThePiachu** [comment]
And this continues to be an issue...
![image](https://user-images.githubusercontent.com/964559/80698863-e975f680-8a8f-11ea-93fb-f46a07539b1c.png)
